test/validation-ui-tests.js

```diff
@@ -625,6 +625,74 @@ QUnit.test('Issue #44 - Validation Element - Is Invalid Test', function(assert)
 
 });
 
+QUnit.test('Validatable can become non-validatable', 0, function(assert) {
+    var vm = {
+        testObj: ko.observable().extend({
+            required: { params: true, message: "oh" }
+        })
+    };
+
+    addTestHtml('<input type="text" id="testElement" data-bind="value: testObj, validationElement: testObj, validationMessage: testObj"/>');
+
+    // make sure we allow element decorations
+    ko.validation.init({ decorateInputElement: true }, true);
+
+    applyTestBindings(vm);
+
+    vm.testObj.extend({validatable: false});
+});
+
+QUnit.test('Issue #519 - validationMessage and validationElement can be applied to non-validatable element', 0, function(assert) {
+    var vm = {
+        testObj: ko.observable()
+    };
+
+    // setup the html
+    addTestHtml('<input type="text" id="testElement" data-bind="value: testObj, validationElement: testObj, validationMessage: testObj"/>');
+
+    // make sure we allow element decorations
+    ko.validation.init({ decorateInputElement: true }, true);
+
+    applyTestBindings(vm);
+});
+
+QUnit.test('Issue #519 - validationMessage and validationElement fields can become unvalidatable', function(assert) {
+    var vm = {
+        testObj: ko.observable().extend({
+            required: true
+        })
+    };
+
+    // setup the html
+    addTestHtml('<input type="text" id="testElement" data-bind="value: testObj, validationElement: testObj, validationMessage: testObj"/>');
```

Comment:
Remove `validationMessage` and `validationElement` bindings as they are automatically applied during data binding - according to configuration specified below. Apart from that, applying `validationMessage` binding to an `input` element doesn't have any benefits.